posix.mak

```diff
@@ -497,24 +490,11 @@ $(foreach p,$(SRC_DOCUMENTABLES),$(eval \
 $(DOC_OUTPUT_DIR)/$(call D2HTML,$p) : $p $(STDDOC) ;\
   $(DDOC) project.ddoc $(STDDOC) -Df$$@ $$<))
 
-html : $(DOC_OUTPUT_DIR)/. $(HTMLS) $(STYLECSS_TGT)
-
-allmod :
-	@echo $(SRC_DOCUMENTABLES)
-
-rsync-prerelease : html
-	rsync -avz $(DOC_OUTPUT_DIR)/ d-programming@digitalmars.com:data/phobos-prerelease/
-	rsync -avz $(WEBSITE_DIR)/ d-programming@digitalmars.com:data/phobos-prerelase/
-
-html_consolidated :
-	$(DDOC) -Df$(DOCSRC)/std_consolidated_header.html $(DOCSRC)/std_consolidated_header.dd
-	$(DDOC) -Df$(DOCSRC)/std_consolidated_footer.html $(DOCSRC)/std_consolidated_footer.dd
-	$(MAKE) -f $(MAKEFILE) DOC_OUTPUT_DIR=$(BIGDOC_OUTPUT_DIR) STDDOC=$(BIGSTDDOC) html -j 8
-	cat $(DOCSRC)/std_consolidated_header.html $(BIGHTMLS)	\
-	$(DOCSRC)/std_consolidated_footer.html > $(DOC_OUTPUT_DIR)/std_consolidated.html
+# this target is called by dlang.org
+html : $(DOC_OUTPUT_DIR)/. $(HTMLS)
 
-changelog.html: changelog.dd
-	$(DMD) -Df$@ $<
+build-html:
```

Comment:
I'm not too happy about this. I would prefer to use `html`, but then we would have to change `dlang.org` to softly detect if a new, special target `html-build` target is existent and wait for the next stable release before we can use just `html` here and then remove the soft fallback at dlang.org afterwards.
Though if wanted we could still do that - it's mostly unrelated to this PR though.

Reply:
My only fear here is if people building Phobos from source have been relying on `make -f posix.mak html` to generate documentation. (I have been. :-P) In any case, if/once we merge this, we should update the wiki Phobos build instructions accordingly.

Reply:
Hmm I don't think the wiki mentions this anywhere. I searched on these pages:

https://wiki.dlang.org/Building_under_Posix
https://wiki.dlang.org/Contributing_to_Phobos
https://wiki.dlang.org/Starting_as_a_Contributor
https://wiki.dlang.org/Contributing_to_dlang.org

Reply:
OK, I may have remembered wrong. In that case, merge ahoy!